### PR TITLE
PMM-7 Fix go-consistent warnings

### PR DESCRIPTION
**Pull request by artemgavrilov**
PMM-7

### CODE REVIEW

File: qan-api2/db.go [hunk 1]
```diff
@@ -133,8 +133,7 @@ func DropOldPartition(db *sqlx.DB, days uint) {
 	err := db.Select(
 		&partitions,
 		query,
-		days,
-	)
+		days)
```
Comment (michal-kralik) [inline qan-api2/db.go:136]: I suggest we get rid of this rule. I think it's easier to navigate the codebase with bracket on new line.

Comment (artemgavrilov) [reply qan-api2/db.go:136]: If you interested in this change you can create poll for BE devs and if majority will prefer bracket on newline then you can rewrite it in all codebase 😁

Comment (recharte) [reply qan-api2/db.go:136]: I agree :+1:

Comment (artemgavrilov) [reply qan-api2/db.go:136]: We have democracy here, go on)

**Review by recharte — APPROVED** [review]

**Pull request merged**